# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/midi/tests/test_midi_send_messages.html
# Commit: 225adb498b09
# Full Hash: 225adb498b091a93ddf7081c1f4356f705594a7b
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-04-12 15:57:11
# Regressor Bug: 1758468
# File Overlap Count: 1
# Description:
#   Bug 1758468 - Use stable, anonymous IDs for Web MIDI ports r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142550
# ==============================================================================

diff -r 3c6e6d0d7b38 -r 225adb498b09 dom/midi/tests/test_midi_send_messages.html
--- a/dom/midi/tests/test_midi_send_messages.html	Tue Apr 12 12:31:27 2022 +0000
+++ b/dom/midi/tests/test_midi_send_messages.html	Tue Apr 12 12:31:27 2022 +0000
@@ -17,7 +17,6 @@
       const access = await navigator.requestMIDIAccess({ sysex: true });
       const output = access.outputs.get(MIDITestUtils.stateTestOutputInfo.id);
 
-      dump("output = " + JSON.stringify(output, null, "  ") + "\n");
 
       // Note on(off).
       output.send([0xff, 0x90, 0x00, 0x00, 0x90, 0x07, 0x00]);
@@ -25,7 +24,6 @@
       try {
         output.send([0x00, 0x01])
       } catch (ex) {
-        dump("Caught exception");
         ok(true, "Caught exception");
       }
 
